### ISSUE TITLE
Methods lost in interfaces

### ISSUE THREAD

**Issue by benjavalero** [jhipster/prettier-java#173]
If an interface has more than one method, only the first one is printed.

Input:
```java
public interface IFacade {

    String getValue();

    String getAnotherValue();

}
```

Output:
```java
 interface IFacade {
String getValue() ;
}
```

**Comment by clement26695** [comment]
Thank you for reporting these issues ! This one should be fixed with 6bcd335 

I'll try to merge today

**Comment by benjavalero** [comment]
Perfect, no methods lost in last version in master branch.
